src/Service.Tests/TestHelper.cs

```diff
@@ -220,103 +232,56 @@ public static RuntimeConfig AddMissingEntitiesToConfig(RuntimeConfig config, str
+        /// </summary>
+        public const string BASE_CONFIG_NULL_NESTED_MUTATIONS_FIELD =
+            RUNTIME_REST_GRAPQL_OPTIONS_CONFIG_JSON +
+              @"
+                ""nested-mutations"": null   
+              }," +
+            HOST_AND_ENTITY_OPTIONS_CONFIG_JSON;
 
         /// <summary>
         /// A minimal valid config json with an empty nested mutations section.
         /// </summary>
         public const string BASE_CONFIG_EMPTY_NESTED_MUTATIONS_FIELD =
-          "{" +
-            SAMPLE_SCHEMA_DATA_SOURCE + "," +
-            @"
-            ""runtime"": {
-              ""rest"": {
-                ""path"": ""/api""
-              },
-              ""graphql"": {
-                ""path"": ""/graphql"",
-                ""allow-introspection"": true,
+
+            RUNTIME_REST_GRAPQL_OPTIONS_CONFIG_JSON +
+              @"
                 ""nested-mutations"": {}
-              },
-              ""host"": {
-                ""mode"": ""development"",
-                ""cors"": {
-                  ""origins"": [""http://localhost:5000""],
-                  ""allow-credentials"": false
-                },
-                ""authentication"": {
-                  ""provider"": ""StaticWebApps""
-                }
-              }
-            },
-            ""entities"": {}" +
-          "}";
+              }," +
+            HOST_AND_ENTITY_OPTIONS_CONFIG_JSON;
 
         /// <summary>
         /// A minimal valid config json with the create field within nested mutation as null.
         /// </summary>
         public const string BASE_CONFIG_NULL_NESTED_CREATE_FIELD =
-          "{" +
-            SAMPLE_SCHEMA_DATA_SOURCE + "," +
-            @"
-            ""runtime"": {
-              ""rest"": {
-                ""path"": ""/api""
-              },
-              ""graphql"": {
-                ""path"": ""/graphql"",
-                ""allow-introspection"": true,
-                 ""nested-mutations"": {
+
+            RUNTIME_REST_GRAPQL_OPTIONS_CONFIG_JSON +
+              @"
+                ""nested-mutations"": {
                       ""create"": null
                  }
-              },
-              ""host"": {
-                ""mode"": ""development"",
-                ""cors"": {
-                  ""origins"": [""http://localhost:5000""],
-                  ""allow-credentials"": false
-                },
-                ""authentication"": {
-                  ""provider"": ""StaticWebApps""
-                }
-              }
-            },
-            ""entities"": {}" +
-          "}";
+              }," +
+            HOST_AND_ENTITY_OPTIONS_CONFIG_JSON;
 
         /// <summary>
         /// A minimal valid config json with an empty create field within nested mutation.
         /// </summary>
         public const string BASE_CONFIG_EMPTY_NESTED_CREATE_FIELD =
-          "{" +
-            SAMPLE_SCHEMA_DATA_SOURCE + "," +
+
+            RUNTIME_REST_GRAPQL_OPTIONS_CONFIG_JSON +
```

Comment:
```suggestion
            RUNTIME_REST_GRAPHQL_OPTIONS_CONFIG_JSON +
```
this should be fixed in all its occurrences